### ISSUE TITLE
Can't find Zik3 device with changed name

### ISSUE THREAD

**Issue by VinciShark** [AeonLucid/Parroter#2]
Windows10 Creator version， already connect zik3 with windows through bluetooth
but can't find Zik3 Device

Should I replace Guid.Parse("8b6814d3-6ce7-4498-9700-9312c1711f64") with something else?

**Comment by AeonLucid** [comment]
Yes, I need to create an array with possible bluetooth GUIDs for parrot zik 3 headphones.

I'll let you know how to find yours soon if you don't know how yet.

**Comment by VinciShark** [comment]
Solved, cause I renamed my Zik3 to `Vinci Zik3 V3`

**Comment by AeonLucid** [comment]
It should work even though the device is renamed.. @VinciShark

**Comment by VinciShark** [comment]
use Service Charactor not name for filting……

btw, any plan working on uwp using javascript? If so, let's build a uwp-js version

**Comment by AeonLucid** [comment]
What is `Service Charactor`?

Also, relevant code for this issue: https://github.com/AeonLucid/Parroter/blob/master/Parroter/Forms/FrmMain.cs#L37

**Comment by VinciShark** [comment]
`private readonly string[] _validParrots = {
            "Parrot ZIK 3 V3.07"
        };`

after rename device, 

`private readonly string[] _validParrots = {
            "Vinci ZIK 3 V3.07"
        };
`
